.github/workflows/continuous-integration.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
           go-version: 1.17
       - name: Run api-docs
         run: |
-          make api-docs
+          make ensure-generate-is-noo api-docs
```

Comment:
This target name looks wrong. But my point was: if you have the `ensure-generate-is-noop` also check the location where the api-docs are expected to be generated, you don't need this extra step, as we check it elsewhere already. Look at the `ci` target.

Reply:
Oh I see. Thanks!